nicegui/elements/table.py

```diff
@@ -2,10 +2,13 @@
 
 from typing_extensions import Literal
 
+from ..dependencies import register_component
 from ..element import Element
 from ..events import TableSelectionEventArguments, handle_event
 from .mixins.filter_element import FilterElement
 
+register_component('ng_table', __file__, 'table.js')
```

Comment:
Why is it called "ng_table"? Is it short for "nicegui_table"? Why not take the full name?

Reply:
Yes, similar to Quasar's `q-input` I chose an abbreviation `ng-` for NiceGUI. But since `ng-` is commonly used with Angular, I'm fine with renaming the tag to `nicegui_table`.

By the way: We currently don't support module names containing dashes "-", because they are used as variable names in JavaScript. But I'll fix that so that we can use the very common kebab case for Vue tags.